modules/edge-identity/src/identity.rs

```diff
@@ -156,17 +158,53 @@ decl_module! {
 				IdentityStage::Attested => ()
 			}
 
+			// Check the number of verifications the record has and ensure all are valid
+			let mut valid_verifications: Vec<(T::AccountId, bool)> = [(_sender.clone(), vote)].to_vec();
```

Comment:
Why do we need a full validity check every call rather than just checking the new one?

Reply:
Verifiers may get voted in or out over time. If this happens, old verifiers votes should not be valid.